cloud/jenkins/psmdb_operator_aks_latest.groovy

```diff
@@ -55,61 +112,10 @@ void prepareNode() {
     }
     echo "USED_PLATFORM_VER=$USED_PLATFORM_VER"
```

Comment:
If we are moving images / versions assignment, maybe we should also move this before Installing tools on the Jenkins executor ?

Reply:
We can't move this specific assignment upper in the script. The reason is that, in order to calculate the USED_PLATFORM_VER, we need to have azure AKS already installed.